src/frontend/lib/DevExperiments.ts

```diff
@@ -0,0 +1,16 @@
+export const devExperiments: { [key: string]: boolean } = {
+  onboarding: process.env.FEATURE_ONBOARDING === "true",
+  appPasscode: process.env.FEATURE_PASSCODE === "true",
+  mapSettings: process.env.FEATURE_MAP_SETTINGS === "true",
+};
+
+const checkFeatureFlagOn = (featureObject: { [key: string]: boolean }) => {
+  for (const k in featureObject) {
+    if (featureObject[k]) {
+      return true;
+    }
+  }
+  return false;
+};
+
+export const featureFlagOn = checkFeatureFlagOn(devExperiments);
```

Comment:
This feels a little more verbose than necessary. Would make this a one-liner:

- using an explicit "or" condition i.e. `devExperiments.onboarding || devExperiments.appPasscode || ...`

- using `Array.some` i.e. `Object.values(devExperiments).some(flag => flag === true)`